course_inventory/inventory.py

```diff
@@ -70,7 +70,9 @@ def make_request_using_api_utils(url: str, params: Union[Dict[str, Any], None] =
                 logger.info('Beginning next attempt')
 
     logger.error('The maximum number of request attempts was reached')
-    return response
+    logger.error(f'Data could not be gathered from the URL with the ending "{url}"')
+    logger.error('The program will exit')
+    sys.exit(1)
```

Comment:
There are already 2 sys.exit's in the code. It might make it easier to debug if we were actually exiting with a different code at each one. Maybe we can have a constant defined somewhere like in vocab.EXIT_CODES?

Cant exit with any code between 1-127 to indicate a failure.

Reply:
Not a bad idea, but aren't the log error messages prior to the exit code just going to be more informative?